cmd/influxd/restore/restore.go

```diff
@@ -416,7 +419,7 @@ func (cmd *Command) uploadShardsPortable() error {
 						targetDB = file.Database
 					}
 
-					if err := cmd.client.UploadShard(file.ShardID, cmd.shardIDMap[file.ShardID], cmd.destinationDatabase, cmd.restoreRetention, tr); err != nil {
+					if err := cmd.client.UploadShard(file.ShardID, cmd.shardIDMap[file.ShardID], targetDB, cmd.restoreRetention, tr); err != nil {
```

Comment:
@aanthony1243 I think this was buggy before, right? We were never actually using `targetDB`.

Reply:
i don't recall any bugs at this level.  any bugs occurring were a step further back in the process.  I will have to take a closer look at this change, but if the tests pass it is probably ok.

Reply:
@e-dard  took a look before sutting down my laptop.  you are right, this is a correct fix.  For better or worse, the client also checks the passed value for "" before it's submitted, so it shouldn't have been a bug.  I may take a closer look later to be sure that the checks are identical but I believe that they are.